tests/overlay_tests.cpp

```diff
@@ -101,11 +101,15 @@ namespace tests {
 constexpr char AGENT_CNI_DIR[] = "cni/";
 constexpr char AGENT_JSON_CONFIG[] = "agent.json";
 constexpr char OVERLAY_SUBNET[] = "192.168.0.0/16";
+constexpr char OVERLAY_SUBNET6[] = "fd02::/64";
 constexpr char OVERLAY_NAME[] = "mz-overlay";
 constexpr char MASTER_JSON_CONFIG[] = "master.json";
 constexpr char MASTER_OVERLAY_MODULE_NAME[] =
   "com_mesosphere_mesos_OverlayMasterManager";
 
+constexpr uint32_t OVERLAY_PREFIX = 24;
+constexpr uint32_t OVERLAY_PREFIX6 = 96;
```

Comment:
For docker bridges the minimum expected prefix is 80 I think? Can you double check.